src/main/java/net/glowstone/util/config/ServerConfig.java

```diff
@@ -412,6 +414,9 @@ private boolean migrate() {
         REGION_CACHE_SIZE("advanced.region-file.cache-size", 256),
         REGION_COMPRESSION("advanced.region-file.compression", true),
         PROFILE_LOOKUP_TIMEOUT("advanced.profile-lookup-timeout", 5),
+        LIBRARY_CHECKSUM_VALIDATION("advanced.library-checksum-validation", true),
+        LIBRARY_REPOSITORY_URL("advanced.library-repository-url",
+                "https://repo.glowstone.net/service/local/repositories/central/content/"),
```

Comment:
More of a question than "please change this", but do we want the default library url to really be our repo? Even though it proxies back to Maven Central, thats still using our bandwidth whenever anyone builds. This question becomes more important when custom libraries are introduced, but I wanted to call it out now.

Reply:
The reason why we use our own repo is because Minecraft Forge was asked to stop using Maven Central when they added library downloading in their installer, so we are trying to avoid that issue by using our own repo.

Reply:
Ah, didn't know about that. Makes sense then.